### PR TITLE
feat(schedule): adaption for support obcloud operation event

### PR DESCRIPTION
**Pull request by abigsaltyfish**
change response, add changeLog for schedule edit

### CODE REVIEW

**Review by yizhouxw — COMMENTED** [review]
PR title should follow developer guide

File: server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/model/ChangeScheduleResponse.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import lombok.Data;
+
+/**
+ * @author jingtian
+ * @date 2024/8/26
+ */
+@Data
+public class ChangeScheduleResponse extends Schedule {
```
Comment (yizhouxw) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/model/ChangeScheduleResponse.java:27]: ChangeScheduleResp, please check RESTful API specfication

File: server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/model/ChangeScheduleResponse.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    @JsonIgnore
+    private ScheduleChangeLog changeLog;
```
Comment (yizhouxw) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/model/ChangeScheduleResponse.java:29]: if @JsonIgnore, then will be skipped while return to client

**Review by yizhouxw — COMMENTED** [review]
please confirm the target branch is correct

File: server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/model/ChangeScheduleResponse.java (hunk 2 above)
Comment (abigsaltyfish) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/model/ChangeScheduleResponse.java]: This field does not require client use

**Review by yizhouxw — APPROVED** [review]
lgtm

**Review by guowl3 — APPROVED** [review]
lgtm

**Pull request merged**